subsys/suit/validator/include/suit_validator.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef SUIT_VALIDATOR_H__
+#define SUIT_VALIDATOR_H__
+
+#include <stdint.h>
+#include <stddef.h>
+#include <suit_plat_err.h>
+
```

Comment:
The `#ifdef __cplusplus` is not strictly necessary, but it i better to be consistent and add it here too :slightly_smiling_face: